components/tag-connect/TC2050-IDC-NL.stanza

```diff
@@ -54,8 +54,8 @@ public pcb-module module :
     jtag-bundle.tck => con.p[4]
     jtag-bundle.tms => con.p[2]
 
-  supports /swd:
-    ;ocdb/bundles/swd.swdclk => con.p[4]
-    ;ocdb/bundles/swd.swdio => con.p[2]
-    /swd.swdclk => con.p[4]
-    /swd.swdio  => con.p[2]
+  supports /swd():
+    ;ocdb/bundles/swd().swdclk => con.p[4]
+    ;ocdb/bundles/swd().swdio => con.p[2]
+    /swd().swdclk => con.p[4]
+    /swd().swdio  => con.p[2]
```

Comment:
The slash is necessary here because a local port was declared called `swd`. 

If you use `swd` without the slash, it will think you're referring to the port called `swd` instead of the bundle called `swd`.